File: server/Checkout Express/src/templates/CxHeader.java
package templates;

public class CxHeader {
	public static String run(Object name, Object logo) {
		return	"<div id=\"'header\"' "+(logo != null ? "class='has-logo'" : "")+">"+
			"\t"+(logo != null ? "<img src='cx_custom/"+logo+"_header.png' />" : "")+""+
			"\t<span>"+(name)+"</span>"+
			"</div>";
	}
}

File: misc/js/templates/cxHeader.js
var template = template || {};

template.cxHeader = function(name, logo) {
	return	"<div id=\"header\" "+(logo != null ? "class='has-logo'" : "")+">\r\n"+
			"\t"+(logo != null ? "<img src='cx_custom/"+logo+"_header.png' />" : "")+"\r\n"+
			"\t<span>"+(name)+"</span>\r\n"+
			"</div>";
};